compiler/compile/OMRNonHelperSymbols.enum

```diff
@@ -256,6 +256,8 @@
     *  The data type of \c <value> indicates the width of the operation.
     */
    atomicFetchAndAddSymbol,
+   atomicFetchAndAdd8BitSymbol,
```

Comment:
Would anyone want 8, 16, 32 and 64 bit versions of `atomicAddSymbol`, which appears above?  It just seems out of place.  Does any downstream project actually use it?

Reply:
My intention was for eclipse-openj9/openj9#18410 to use these new symbols as it insufficient to just ask if the code generator can support the `atomicAddSymbol` without knowing what the type of operand would be. If that particular type is unsupported on a specific architecture, we would need to fall back to equivalent software implementation.

Reply:
My question might not have been clear.  You've made changes so that OMR will have 8, 16, 32 and 64 bit versions of `atomicFetchAndAdd*BitSymbol` and `atomicSwap*BitSymbol` non-helpers.  In the case of the very similar `atomicAddSymbol`, we only have the generic version.  Is there any reason why this change should not introduce versions of that non-helper for all the different bit sizes, just for consistency?